docs/management/admin/event-filters.asciidoc

```diff
@@ -33,24 +33,26 @@ image::images/event-filter.png[]
 [discrete]
 == View and manage event filters
 
-The Event filters list allows you to view and manage all endpoint event filters that have been added. To view the Event filters list, go to *Manage* -> *Event filters*. Event filters appear in reverse chronological order, with the most recently modified filter at the top. Each filter has its own entry, which displays details such as the filter name, operating system, date created, and the filter conditions.
+The Event filters list allows you to view and manage all endpoint event filters that have been added. To view the Event filters list, go to *Manage* -> *Event filters*. Event filters appear in reverse chronological order, with the most recently created filter at the top. Each filter has its own entry, which displays details such as the filter name, operating system, date created, and the filter conditions.
+
+To refine the Event filters list, enter a query in the search bar. You can search the by name, comments, or the value of a field.
 
 [role="screenshot"]
 image::images/event-filters-list.png[]
 
-To refine the Event filters list by a specific value, enter a query in the https://elastic.github.io/eui/#/forms/search-bar[search bar]. You can search the following attributes:
-
-* `name`
-* `comments`
-* `value`
-
+[discrete]
+[[edit-event-filter]]
+=== Edit an event filter
 To edit an event filter:
 
-. Click *Edit* on the appropriate event filter card.
-. Modify details or conditions as appropriate.
-. Click *Update event filter*. The newly modified filter appears at the top.
+. Click the actions button (*...*) for the event filter you want to edit, then select *Edit event filter*.
```

Comment:
Should we cap the button title, i.e., `Actions button` even if there isn't a UI label?

Reply:
I'd prefer lowercase if there isn't a UI label or any other specific reason to capitalize.